hardware/tests/test_benchmark_mem.py

```diff
@@ -98,17 +98,6 @@ def test_mem_perf_bytes(self, mock_popen, mock_cpu_socket,
         expected = EXPECTED_RESULT
         self.assertEqual(sorted(expected), sorted(self.hw_data))
 
-    def test_mem_perf_text(self, mock_popen, mock_cpu_socket,
```

Comment:
ditto

Reply:
because the output of commands in Python 3.x is bytes, not pure text, so we don't need this, we already have the correspondent bytes test